src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -395,6 +396,12 @@ fn make_data(
         data.insert("livereload".to_owned(), json!(livereload));
     }
 
+    let default_theme = match html_config.default_theme {
```

Comment:
Just out of curiosity, what's going to happen if I set the `default-theme` value in my `book.toml` to something which isn't a theme?

Reply:
It will be accepted and will be used as the class on the body and the value for the global `default_theme` js variable. This will cause the book to not look quite right in certain places, though for the most part the default styling does a good job.

The reason for not validating this further is that someone could create their own theme with an option that the default theme doesn't have, so there is no real way to check the validity other than checking if a value is specified.